scripts/openshiftci-presubmit-integratione2ebenchmark.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+export CI="openshift"
+export TIMEOUT="30m"
+export GINKGO_VERBOSE_MODE="-v"
+make configure-installer-tests-cluster
+make bin
+mkdir -p $GOPATH/bin
+go get -u github.com/onsi/ginkgo/ginkgo
+export PATH="$PATH:$(pwd):$GOPATH/bin"
+export ARTIFACTS_DIR="/tmp/artifacts"
+export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+
+# Integration tests
+make test-generic
```

Comment:
Wouldn't it be better to run `make test integration` it will be also safer as this will make sure that we don't forget any test.

Reply:
Right